internal/broker/bind_create_test.go

```diff
@@ -234,7 +192,7 @@ func TestCreateBindingEndpoint(t *testing.T) {
 
 		//// verify connectivity using kubeconfig from the generated binding
 		newClient := kubeconfigClient(t, binding.Credentials.(string))
-		_, err = newClient.CoreV1().Secrets("default").Get(context.Background(), "secret-to-check", v1.GetOptions{})
+		_, err = newClient.CoreV1().Secrets("kyma-system").Get(context.Background(), "secret-to-check", v1.GetOptions{})
 		assert.NoError(t, err)
```

Comment:
Could we please add a test to check if intended service accounts, roles and bindings exist?